frontend/src/components/lib/components/Logo/Logo.tsx

```diff
@@ -18,6 +19,7 @@ export const Logo = (props: LogoProps) => {
   const logo = require('../../assets/logo/logo.svg').default;
   const logoBlack = require('../../assets/logo/logo-black.svg').default;
   const logoWhite = require('../../assets/logo/logo-white.svg').default;
+  const logoMon = require('../../assets/logo/monitoring.svg').default;
```

Comment:
the word "logo" should appear in the filename, "monitoring.svg" could be anything